src/controllers/auth.controller.ts

```diff
@@ -56,3 +73,299 @@ export async function signupHandler(
 		});
 	}
 }
+
+export async function verifyUserHandler(
+	req: Request<VerifyUserSchema["params"]>,
+	res: Response
+) {
+	const { email, verificationCode } = req.params;
+
+	try {
+		// QUESTION: should we use username to find the user or email?
```

Comment:
In the verification, Keep it Email! For the dash, keep it both "Username and Email."

Reply:
Ok